lib/fauna/query.rb

```diff
@@ -1,4 +1,7 @@
 module Fauna
+
+  extend self
```

Comment:
Arguably, the latter is not necessary, since client.query is more useful, will probably be more used.